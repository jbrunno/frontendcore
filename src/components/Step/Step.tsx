import React from 'react';
import { Step as MuiStep, StepProps as MuiStepProps } from '@mui/material';

export type StepProps = MuiStepProps;

export function Step(props: StepProps) {
  return <MuiStep {...props} />;
}

export default Step;
