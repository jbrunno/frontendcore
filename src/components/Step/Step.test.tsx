import React from 'react';
import { render } from '@test-utils/component-testing';
import { Step } from './Step';

test('renders the Step', () => {
  const { container } = render(<Step />);
  expect(container).toMatchSnapshot();
});
