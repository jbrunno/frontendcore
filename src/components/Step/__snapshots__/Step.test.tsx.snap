// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Step 1`] = `
<div>
  <div
    class="MuiStep-root emotion-0"
  />
</div>
`;
